nss-cohort-36/bangazon-api-virtual_magicians
# Jl order apiside

# Description
[Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.]
Closes #[Enter issue #]

## Type of change
Please delete options that are not relevant.

# Testing Instructions
Make sure to commit local changes in your branch before switching to master.
1. `git checkout master`
2. `git fetch --all`
3. `git checkout order-apiside`
4. will need to re run migrations to have tables up to date with correct naming 
5. pull react request and go to buy in navbar - 

# Checklist:
